src/Servers/Kestrel/Transport.MsQuic/README.md

```diff
@@ -0,0 +1,9 @@
+## Using MsQuic on Windows
+
+### Setup pre-requisites
+
+1. Update machine to the latest Windows Insider build. This is required for TLS 1.3 support.
```

Comment:
Minimum version number? Insiders has several tracks.

Reply:
No idea, but I'll recommend the Fast track.

Reply:
Check the current build number in the fast track and put it in as a minimum required build number. Here's a suggestion but you'll need to fill in the correct build number before accepting:

```suggestion
1. Update machine to the latest Windows Insiders build (build number 12345.6 or later). This is required for TLS 1.3 support.
```

Reply:
I'm not on the fastest insider build; I have a custom one that targets a branch. I can try to figure it out...